spec/rails_helper.rb

```diff
@@ -30,6 +30,17 @@
   config.after type: :request do
     logout
   end
+
+  config.around(:each, testing_transactions: true) do |example|
+    DatabaseCleaner.allow_remote_database_url = true
+    self.use_transactional_tests = false
+    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.clean
+    example.run
+    DatabaseCleaner.clean
+    DatabaseCleaner.strategy = :transaction
```

Comment:
`DatabaseCleaner.strategy = :transaction` shouldn't be necessary here, as DatabaseCleaner isn't being used for other tests, right?

Reply:
You're right, I'll remove that